From lethal@linux-sh.org Sun Feb 19 13:08:33 2006
Date: Sun, 19 Feb 2006 23:08:32 +0200
From: Paul Mundt <lethal@linux-sh.org>
To: Greg KH <greg@kroah.com>, Christoph Hellwig <hch@infradead.org>, <zanussi@us.ibm.com>
Subject: [PATCH 3/5] sysfs: Update relay file support for generic relay API.
Message-ID: <20060219210832.GD3682@linux-sh.org>
Content-Disposition: inline

Now with the introduction of CONFIG_REPLAY, switch to using that instead
of CONFIG_REPLAYFS_FS.

Signed-off-by: Paul Mundt <lethal@linux-sh.org>

---
 fs/sysfs/Makefile     |    2 +-
 fs/sysfs/dir.c        |    2 +-
 include/linux/sysfs.h |    4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

--- gregkh-2.6.orig/fs/sysfs/Makefile
+++ gregkh-2.6/fs/sysfs/Makefile
@@ -4,4 +4,4 @@
 
 obj-y		:= inode.o file.o dir.o symlink.o mount.o bin.o \
 		   group.o
-obj-$(CONFIG_RELAYFS_FS) += relay.o
+obj-$(CONFIG_RELAY) += relay.o
--- gregkh-2.6.orig/fs/sysfs/dir.c
+++ gregkh-2.6/fs/sysfs/dir.c
@@ -88,7 +88,7 @@ static int init_file(struct inode * inod
 	return 0;
 }
 
-#ifdef CONFIG_RELAYFS_FS
+#ifdef CONFIG_RELAY
 static int init_relay_file(struct inode * inode)
 {
 	extern struct file_operations relay_file_operations;
--- gregkh-2.6.orig/include/linux/sysfs.h
+++ gregkh-2.6/include/linux/sysfs.h
@@ -95,7 +95,7 @@ struct sysfs_dirent {
 #define SYSFS_NOT_PINNED	(SYSFS_KOBJ_ATTR | SYSFS_KOBJ_BIN_ATTR | \
 				 SYSFS_KOBJ_LINK | SYSFS_KOBJ_RELAY_ATTR)
 
-#if defined(CONFIG_RELAYFS_FS) && defined(CONFIG_SYSFS)
+#if defined(CONFIG_RELAY) && defined(CONFIG_SYSFS)
 int sysfs_create_relay_file(struct kobject *, struct relay_attribute *);
 void sysfs_remove_relay_file(struct kobject *, struct relay_attribute *);
 #else
@@ -107,7 +107,7 @@ static inline int sysfs_create_relay_fil
 static inline void sysfs_remove_relay_file(struct kobject *kobj, struct relay_attribute *attr)
 {
 }
-#endif /* CONFIG_RELAYFS_FS && CONFIG_SYSFS */
+#endif /* CONFIG_RELAY && CONFIG_SYSFS */
 
 #ifdef CONFIG_SYSFS
 
